backend/src/main/java/com/woowacourse/kkogkkog/exception/GlobalControllerAdvice.java

```diff
@@ -35,6 +42,17 @@ public ResponseEntity<ErrorResponse> handleException(ForbiddenException e) {
         return new ResponseEntity<>(new ErrorResponse(e.getMessage()), HttpStatus.FORBIDDEN);
     }
 
+    @ExceptionHandler
+    public ResponseEntity<ErrorResponse> handleInvalidRequest(
+        MethodArgumentNotValidException bindingResult) {
+        String causes = bindingResult.getFieldErrors()
+            .stream()
+            .map(DefaultMessageSourceResolvable::getDefaultMessage)
+            .collect(Collectors.joining(INVALID_REQUEST_DELIMITER));
+        String errorMessage = String.format(INVALID_REQUEST_EXCEPTION_MESSAGE_FORMAT, causes);
+        return ResponseEntity.status(HttpStatus.BAD_REQUEST).body(new ErrorResponse(errorMessage));
+    }
```

Comment:
위의 exception handler 메서드들의 반환하는 코드와 통일시켜주면 좋을 것 같아요 :)

```suggestion
        return new ResponseEntity<>(new ErrorResponse(e.getMessage()), HttpStatus.BAD_REQUEST);
    }
```

Reply:
넵 반영했습니다.